conceptual/Npgsql/release-notes/8.0.md

```diff
@@ -1,29 +1,33 @@
 # Npgsql 8.0 Release Notes
 
-Npgsql version 8.0 is under development, and is available as preview versions.
+**The below release notes and breaking changes list aren't yet complete. Check back soon for more notes.**
+
+Npgsql version 8.0 is under development, and is available as release candidate versions.
 
 ## New features
```

Comment:
We should mention nullable parameter support in copy and parameters
https://github.com/npgsql/npgsql/issues/1965
https://github.com/npgsql/npgsql/issues/3679

There is also some info that needs changes here, as we now do support sending nulls
https://www.npgsql.org/doc/basic-usage.html#strongly-typed-parameters

Reply:
We can decide to mention something about the polymorphic json support https://github.com/npgsql/npgsql/issues/4862

Reply:
Oh absolutely, this is just a first round - we definitely need a lot more release notes; I'll be working on that (putting your suggestions in my notes).